script/fossa.mk

```diff
@@ -0,0 +1,17 @@
+# Variables for Fossa
+BUILD_ANALYZER?=dockereng/fossa-analyzer
+FOSSA_OPTS?=--option all-tags:true --option allow-unresolved:true
```

Comment:
Does fossa need a "strategy" set for python? https://github.com/fossas/fossa-cli/blob/master/docs/integrations/python.md#python